axelor-studio/src/main/webapp/custom-model/static/js/main.js

```diff
@@ -41,7 +41,7 @@ if("border-box"===d?h+=l:"content-box"===d&&(h-=u),null!==r||null!==o){s.value="
 o=[],a=[]):(a=this.allocateChildren(o,a),this.checkEmptyColumns(a,i.nodeId),i.nodeData.children=a,t[0].push(i),i={},o=[],a=[]);var d=JSON.parse(u.widgetAttrs),c=Number(null!==d&&d.itemSpan?d.itemSpan:6);a=this.createChildColumns(12/c),i={nodeId:n,column:Number(null!==d&&d.itemSpan?d.col:1),isExists:!0,nodeData:{type:g.default.CONTAINER,properties:u,children:[]}}}n++}return r?(r=!1,t.push(o),o=[],a=[]):(a=this.allocateChildren(o,a),this.checkEmptyColumns(a,i.nodeId),i.nodeData.children=a,t[0].push(i),i={},o=[],a=[]),t[0].push(this.getDummyNode("root")),{fieldData:t,counter:n}}},{key:"componentDidMount",value:function(){this.getModelList(),this.fetchAppBuilderList(),this.createNewForm(),window.addEventListener("resize",this.updateWindowDimensions),this.updateWindowDimensions(),this.saveListner()}},{key:"saveListner",value:function(){var e=this;document.addEventListener("keydown",function(t){for(var n=void 0,r=[t.key,t.keyIdentifier,t.keyCode,t.which];void 0===n&&r.length>0;)n=r.pop();return!(n&&(""+n=="115"||""+n=="83")&&(t.ctrlKey||t.metaKey)&&!t.altKey)||(t.preventDefault(),document.activeElement.blur(),e.isSaveDisable()||e.saveFormData(),!1)})}},{key:"fetchAppBuilderList",value:function(){var e=this,t=new O.default;t.search("com.axelor.studio.db.AppBuilder",[],0,null,["id"],["id","name"]).then(function(t){var n=t.data,r=void 0===n?[]:n;return e.setState({appBuilderL
```

Comment:
You mustn't update this file. This file is used to allow user to select a fa-icon on studio. Now we have two icons fa-google but not more the fa-google-plus icon